app/Community/Controllers/UserSettingsController.php

```diff
@@ -54,7 +57,18 @@ public function show(): InertiaResponse
             'updateMotto'
         );
 
-        $props = new UserSettingsPagePropsData($userSettings, $can);
+        /** @var Collection<int, Role> $displayableRoles */
+        $displayableRoles = $user->displayableRoles()->orderBy('display')->orderBy('name')->get();
```

Comment:
Would it make sense to just order these alphabetically instead of by display order? When I first saw the dropdown I assumed it wasn't sorted at all.

Reply:
Yeah, I like that approach more. Updated in latest.